src/tests/sentry-mocked.test.ts

```diff
@@ -91,6 +91,7 @@ Object {
   "data": "fatal",
   "scope": Object {
     "extras": Object {
+      "cosmas.pkgVersion": "2.0.0-rc.2",
```

Comment:
We will have to do something about this or we will need to change on every rc rls! :D

Reply:
Fixed in cae1000426e51ce61979a47736e73bceffb0f9ea